pkg/controller/seed-controller-manager/mla/org_grafana_controller.go

```diff
@@ -221,12 +255,29 @@ func (r *orgGrafanaController) createGrafanaOrg(ctx context.Context, expected gr
 	return expected, nil
 }
 
-func (r *orgGrafanaController) ensureOrganization(ctx context.Context, log *zap.SugaredLogger, project *kubermaticv1.Project, expected grafanasdk.Org, annotationKey string) error {
+func (r *orgGrafanaController) ensureDashboards(ctx context.Context, log *zap.SugaredLogger, orgID uint) error {
+	configMapList := &corev1.ConfigMapList{}
+	listOptions := &ctrlruntimeclient.ListOptions{Namespace: r.mlaNamespace}
+	if err := r.List(ctx, configMapList, listOptions); err != nil {
```

Comment:
actually, there is a helper function to list objects in namespace, and you can do something like:
```go
if err := r.List(ctx, configMapList, ctrlruntimeclient.InNamespace(r.mlaNamespace)); err != nil {
}
```

Reply:
oh, cool